abci/backend/tendermint.go

```diff
@@ -403,16 +403,19 @@ func (b *TendermintBackend) Start(app abci.Application) error {
 			return err
 		}
 
-		fnConsensusReactor, err := CreateFnConsensusReactor(b.OverrideCfg.ChainID, privVal, b.FnRegistry, cfg, nodeLogger,
-			dbProvider, b.OverrideCfg.FnConsensusReactorConfig)
-		if err != nil {
-			return err
+		reactorConfig := b.OverrideCfg.FnConsensusReactorConfig
+		if reactorConfig.IsValidator {
```

Comment:
If you don't create the reactor then non-validators won't forward the messages to the validators.